cypress/integration/south_america_spec.js

```diff
@@ -0,0 +1,67 @@
+describe('South America game', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.get('.nav-link').contains('South America').click()
+  })
+  it('should display a bingo grid', () => {
+    cy.get('.board');
+    cy.get('.square').should('have.length', 16)
+  })
+  it('should have an area for displaying coordinates', () => {
+    cy.get('.coordinates').contains('N/S');
+    cy.get('.coordinates').contains('E/W');
+    cy.get('.coordinates').contains('Get Coordinates');
+  })
+  it('should have a button for setting the game board', () => {
+    cy.get('.bingo-btn').contains('Set Game Board!');
+  })
+  it('should have a map displaying the region', () => {
+    cy.get('.leaflet-container')
+  })
+  it('should be able to populate the game board with coordinates', () => {
```

Comment:
Since the game board countries are randomized, instead of checking the specific countries, we could test that 
1) each square has a name property with a length !== 0
2) none of the square names are the same
3) each square (or test maybe one or two) is from the appropriate region

This can be fleshed out in a later merge